### PR TITLE
Hindi Translations Added under config/locales/hi

### LINKED ISSUE #431: Hindi Translation
Hi there,

I am willing to add Hindi translation for the website based on @DeeDeeG & @Intimaria working for Spanish translation.

I am more than happy to do my little part for the community in India & folks who will be using it while travelling.

Cheers,
Ashish

### PR DESCRIPTION
**Pull request by sankalpchauhan-me**
# Context
- Fixes #431 
- I have added Hindi translations to all the files present in config/locales/en then I saved the translations to a new folder at config/locales/hi

# Summary of Changes

- All the translations are correct to the best of my knowledge (I am a native Hindi speaker and have full professional proficiency in English)
- All files are translated by me except config/locales/hi/devise.hi.yml for this particular file I have used the original source mentioned in the file i.e. https://github.com/plataformatec/devise/wiki/I18n
- I have not translated commented lines in documents, as they won't be visible in the actual app

# Checklist
- **No changes were made to existing code base total 14 files added in config/locales/hi** 
-  I have changed the top line of each file from en: -> hi: as mentioned by @DeeDeeG in Issue #431 
- I have renamed every file from filename.en.yml to filename.hi.yml as  mentioned by @DeeDeeG in Issue #431 . except for the file buisness_info.yml as it did not contain the keyword 'en'.
- Translated only to the right of a colon : from English into Hindi.
         -hello: 'hello!' --> hello: 'नमस्ते!' (as  mentioned by @DeeDeeG in Issue #431)

- [ ] Tested Mobile Responsiveness
- [ ] Added Unit Tests
- [x] CI Passes
- [ ] Deploys to Heroku on test Correctly (Maintainers will handle)
- [ ] Added Documentation (Service and Code when required)

# Screenshots
N/A

### CODE REVIEW

**Comment by DeeDeeG** [comment]
Wow, @sankalpchauhan-me thank you for doing this! It has been a while since that issue was posted, I was almost expecting that we might never receive this translation.

I will take a look at the error message at Travis and see what might be the issue, but overall these translations look very good.

Usually we like to have a second native speaker look over any new translation to be sure it is correct. We can look for another person who might be able to review this. Also, if you know someone who might be willing to review and give feedback on the translations, it might speed up the time it takes to review this PR.

Thanks again!

File: config/locales/hi/restroom.hi.yml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+      positive: '%{percentage}% सकारात्मक'
+      unrated: 'अभी तक मूल्यांकित नहीं'
+    upvote: 'अच्छा हैं !'
+    downvote: 'सुरक्षित नहीं'
+    confirm:
+      upvote: 'क्या आप वाकई इस रेस्टरूम को अच्छी रेटिंग देना चाहते हैं?'
+      downvote: 'क्या आप वाकई इस रेस्टरूम को ख़राब रेटिंग देना चाहते हैं?'
+    flash:
+      field: 'एक आवश्यक फ़ील्ड खाली छोड़ दिया गया है। कृपया अपना सबमिशन पुनः जांचें और फिर फॉर्म सबमिट करें।'
+      unexpected: 'हम एक अप्रत्याशित समस्या में आ  गए, अगर समस्या जारी रहती है तो कृपया हमसे संपर्क करें।
```
Comment (DeeDeeG) [inline config/locales/hi/restroom.hi.yml]: This line is missing a closing quotation mark `'`.

Everything else looks good!

Comment (DeeDeeG) [reply config/locales/hi/restroom.hi.yml]: This was the error message, if anyone is particularly interested:

> I18n::InvalidLocaleData: can not load translations from /refugerestrooms/config/locales/hi/restroom.hi.yml: #<Psych::SyntaxError: (/refugerestrooms/config/locales/hi/restroom.hi.yml): did not find expected key while parsing a block mapping at line 42 column 7>

https://travis-ci.org/RefugeRestrooms/refugerestrooms/builds/436778300#L1888

**Comment by sankalpchauhan-me** [comment]
I have corrected the errors reported by Travis CI, the build is now passing.

**Comment by DeeDeeG** [comment]
Thank you very much!

All that is left is to find another native speaker of Hindi, so we can review the actual translations. Once such a review has happened, we would be glad to merge. (If for some reason we can't find a reviewer for a long time, we may review using Google Translate just to make sure this can go live at refugerestrooms.org, rather than sitting as a Pull Request forever.)

We appreciate this contribution very much. And happy Hacktoberfest! :jack_o_lantern: :computer: :fallen_leaf: :sparkles:

**Comment by sankalpchauhan-me** [comment]
Happy Hacktoberfest to you too @DeeDeeG

**Comment by mi-wood** [comment]
I put out a tweet to see if someone can review this: https://twitter.com/REFUGErestrooms/status/1049416630604689410

**Comment by DeeDeeG** [comment]
Thank you, @mi-wood!

**Comment by DeeDeeG** [comment]
Hi @ashishwadekar,

It has been some time since you initially submitted issue #431 to our repository, and someone has now submitted a Hindi translation for Refuge Restrooms. Would you be interested in reviewing this translation for accuracy, and suggesting any improvements you think it needs?

Best regards,

\- DeeDeeG

**Comment by ashishwadekar** [comment]
Hi @DeeDeeG 

Sure, I am more than happy to review translation and post my suggestions here.

Cheers,
Ashish

**Comment by ashishwadekar** [comment]
Hi @sankalpchauhan-me @DeeDeeG 

The translation looks good to me. बोहोत हि उम्दा काम किया है अपने (You have done a fantastic job)

@sankalpchauhan-me few suggestions to make translation better:

संसाधन पन्ना 
साइन्स -> प्रतिक

बारे में पन्ना  
सरलपुर्वक -> सरलतापूर्वक
बच्चे -> व्यक्ति 

Rest all is OK.

Cheers,
Ashish

**Comment by DeeDeeG** [comment]
Thank you for the review, @ashishwadekar!

**Comment by sankalpchauhan-me** [comment]
@ashishwadekar @DeeDeeG I have committed the required changes.

**Comment by DeeDeeG** [comment]
Thanks @sankalpchauhan-me! I should be able to take a look at this within the next couple of days and merge.

**Comment by DeeDeeG** [comment]
I "deployed" a copy of this branch, so anyone can take a look at it, here:
https://refuge-cedar-14.herokuapp.com/

And here is a video capture of what it looks like, also:
[refugerestrooms-hindi-translation-mp4.zip](https://github.com/RefugeRestrooms/refugerestrooms/files/2577522/refugerestrooms-hindi-translation-mp4.zip)

**Comment by DeeDeeG** [comment]
Alright, it is merged now;,thanks very much to both of you, @sankalpchauhan-me and @ashishwadekar!

**Comment by sankalpchauhan-me** [comment]
Everything looks good in the deployment, as the PR is merged I am deleting my copy of the branch i.e.sankalpchauhan-me:hindi-translations-develop, I will keep contributing to the project in future.
Best Of Luck to everyone working on this project.

**Comment by sankalpchauhan-me** [comment]
I am a self taught Native Android Developer currently pursuing my Bachelors of Technology in Information Technology @DeeDeeG if there is some Native Android App of this project, I could contribute to that.

**Comment by DeeDeeG** [comment]
We would be happy to have contributions over at out Android app, here: https://github.com/RefugeRestrooms/refugerestrooms-android/ !

We might not be able to review pull requests right away, as our maintainers there are somewhat busy, but contributions are still appreciated, and hopefully we can get to them soon.

**Pull request merged**